pkg/engine/artifacts.go

```diff
@@ -140,6 +140,12 @@ func kubernetesContainerAddonSettingsInit(p *api.Properties) map[string]kubernet
 			destinationFile: "azure-npm-daemonset.yaml",
 			isEnabled:       k.IsAddonEnabled(AzureNetworkPolicyAddonName),
 		},
+		AzurePolicyAddonName: {
+			sourceFile:      "kubernetesmasteraddons-azure-policy-deployment.yaml",
```

Comment:
nit: let's keep a concise name for this new addon spec: `azure-policy-deployment.yaml`

Reply:
thanks, renamed